src/system/hardware.cpp

```diff
@@ -58,15 +58,21 @@ void setupHardwarePins()
   //pinMode(BLE_COMMAND_MODE_PIN, OUTPUT); // Command Mode pin for BLE
   
   pinMode(INTERRUPT_LINE_7_PIN, INPUT_PULLUP); // This the interrupt line 7
+  
+  // Moved to generic_analog driver for INPUT_ANALOG mode
+  // Need to test and see if these pins can also be used as GPIO for actuator drivers
+  /*
   pinMode(ANALOG_INPUT_1_PIN, INPUT_ANALOG);
   pinMode(ANALOG_INPUT_2_PIN, INPUT_ANALOG);
   pinMode(ANALOG_INPUT_3_PIN, INPUT_ANALOG);
   pinMode(ANALOG_INPUT_4_PIN, INPUT_ANALOG);
   pinMode(ANALOG_INPUT_5_PIN, INPUT_ANALOG);
+  */
   pinMode(ONBOARD_LED_PIN, OUTPUT); // This is the onboard LED ? Turns out this is also the SPI1 clock.  niiiiice.
 
-  pinMode(GPIO_PIN_3, OUTPUT); // GPIO pin available
-  pinMode(GPIO_PIN_4, OUTPUT); // in use for 5v booster
+  pinMode(GPIO_PIN_6, OUTPUT); // GPIO pin available
+  pinMode(GPIO_PIN_3, OUTPUT); // in use for 5v booster
+  // pinMode(GPIO_PIN_5, OUTPUT); // in use for DHT22 [INPUT_PULLUP]
```

Comment:
Can this GPIO setup be moved into the drivers?

Reply:
These GPIO Pins are for hardware things like the 5v booster and then for the actuator class that Anya is working on. I was going to ask if we want to have the 5v booster included into the actuator class, or if we should have an additional hardware class. I think you were going to use #3 for the 5v booster you were incorporating into the next version? So that pin setup would probably move into more specific usage.

Reply:
Would that mean having 4 pins per port instead of 3?  Would the power pin be different from the vdd rails currently?

Reply:
Interesting.  I guess there's a limit to how multi-use we should make these ports.  It would be confusing to supply both AVREF and +5V on the same port.  We'll have to keep thinking about how we might make this more generic.